README.md

```diff
@@ -0,0 +1,116 @@
+there combination into spherical grids.
+
+## Design Goals 
+
+* Provide a stable and reproducible implementation of the various atomic and
+molecular grids commonly encountered in quantum chemistry calculations
+* Develop a modern, modular, extensible C++ API to allow for the implementation
+and validation new atomic and molecular quadrature schemes.
+
+## Dependencies
+
+* CMake (3.17+)
+* Modern C++ compiler (C++17 compliant)
+
+## Major Contributors
+
+* David Williams-Young - LBNL (dbwy at lbl dot gov)
+* Susi Lehtola - University of Helsinki
+
+## Implemented Quadratures
+
+A note on quadrature classifications:
+* Primitive quadratures are those generated on a finite bound (e.g. Gauss quadrature rules). The general software design pattern of IntegratorXX is to build up higher-order quadrature rules (e.g. radial transformation, etc) from these primitive quadratures.
+* Radial quadratures are convolutions of primitive quadrature rules with a radial transformation scheme (mapping the natural domain of the primitive quadrature to positive semi-indefinite). The jacobian of the transformation *is* included while the radial component of the spherical jacobian is *not*
+* Angular quadratures integrate over $S^2$ (solid angle). It is typically the case that these are manually constructed to integrate spherical harmonics up to a specific order, and are thus only compatible witch *specific* grid orders.
+
+
+Below are the quadratures currently implemented in IntegratorXX. Please refer to the
+source for appropriate references.
+
+| Quadrature Name                 | Quadrature Class | Domain  | C++ Class           |
+|---------------------------------|------------------|---------|---------------------|
+| Gauss-Chebyshev (First Kind)    | Primitive        | (-1,1)  | `GaussChebyshev1`   |
+| Gauss-Chebyshev (Second Kind)   | Primitive        | [-1,1]  | `GaussChebyshev2`   |
+| Gauss-Chebyshev (Third Kind)    | Primitive        | (0,1)   | `GaussChebyshev3`   |
+| Gauss-Legendre                  | Primitive        | [-1,1]  | `GaussLegendre`     |
+| Gauss-Lobatto                   | Primitive        | [-1,1]  | `GaussLobatto`      |
+| Trapezoid Rule                  | Primitive        | [0,1]   | `UniformTrapezoid`  |
+| Becke                           | Radial           | (0,inf) | `Becke`             |
+| Murray-Handy-Laming (MHL)       | Radial           | (0,inf) | `MurrayHandyLaming` |
+| Mura-Knowles (MK)               | Radial           | (0,inf) | `MuraKnowles`       |
+| Treutler-Ahlrichs (TA, M3 + M4) | Radial           | (0,inf) | `TreutlerAhlrichs`  |
+| Ahrens-Beylkin                  | Angular          | $S^2$   | `AhrensBeylkin`     |
+| Delley                          | Angular          | $S^2$   | `Delley`            |
+| Lebedev-Laikov                  | Angular          | $S^2$   | `LebedevLaikov`     |
+| Womersley                       | Angular          | $S^2$   | `Womersley`         |
```

Comment:
I would split this table in three: primitive, radial, and angular quadratures, which could be discussed in subsections.

Reply:
Agreed. Fixed